src/containers/ManageListingsPage/ManageListingsPage.duck.js

```diff
@@ -143,6 +166,30 @@ const manageListingsPageReducer = (state = initialState, action = {}) => {
       };
     }
 
+    case DISCARD_DRAFT_REQUEST:
+      return {
+        ...state,
+        discardingDraft: payload.listingId,
+        discardingDraftError: null,
+      };
+    case DISCARD_DRAFT_SUCCESS:
+      return {
+        ...removeDiscardedDraftListing(state, payload.data),
```

Comment:
I don't think this is needed anymore 🤔

Reply:
It still clears the `discardingDraft` id from state even after the removeDiscardedDraftListing function is removed, so I'd keep it here.

Reply:
I meant just the `removeDiscardedDraftListing` function.